modules/arm_plugin/thirdparty/external_kernels/README.md

```diff
@@ -0,0 +1,5 @@
+# List of external files that used in [OpenVINO ARM plugin](https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/arm_plugin)
+
+| **Original link on file**                                                                    | **Original repo** |
+|----------------------------------------------------------------------------------------------|-------------------|
+| [ncnn/neon_math.h](https://github.com/Tencent/ncnn/blob/master/src/layer/arm/neon_mathfun.h) | [NCNN](https://github.com/Tencent/ncnn) |
```

Comment:
can we provide de a full link including commit hash ?

Reply:
@ilya-lavrenov done